resources/tests.mjs

```diff
@@ -132,6 +132,35 @@ Suites.push({
     ],
 });
 
+Suites.push({
+    name: "TodoMVC-React-Big",
```

Comment:
"Big" is a vague term. How about "Complex-DOM"?

Reply:
I don't disagree (and don't oppose a name change) but my understanding of the longer term plan based on the working group discussions is that this test will be removed and so the name isn't too important. Instead, the complex DOM will ultimately get absorbed into the existing todo workloads (example how this might look in https://github.com/WebKit/Speedometer/pull/150#issuecomment-1535307111). Does that match your thinking?

There are of course sequencing questions about if that should happen now (i.e. close this PR once everyone is happy with the code and make a new one that lands the generator into resources/todomvc and modifies TodoMVC-React) or later (land this and then make a PR that moves the files around and deletes the tentative test).